ui/pages/permissions-connect/snaps/snap-result/snap-result.js

```diff
@@ -108,15 +103,15 @@ export default function SnapResult({
       case 'wallet_installSnap':
         failedScreenTitle = t('snapInstallationErrorTitle');
         failedScreenDescription = t('snapInstallationErrorDescription', [
-          <Text as={ValidTag.Span} key="1" fontWeight={FontWeight.Medium}>
```

Comment:
why are we no longer using the enum?

Reply:
Good question @hmalik88! We deprecated `ValidTag` in favor of a string union type so the Text `as` prop was consistent with other components `as` prop type(`React.ElementType`). The `Text` component was the only component that limited the elements it could render with an enum. Did you prefer using the `ValidTag` enum?

Reply:
Is the aim to be able to use both a string and an enum? If so you can use a type like this:

```export type EnumToUnion<Type extends string> = `${Type}`;```

Reply:
And yes, I would prefer enum usage -- the above lets you have both.

Reply:
Ok good to know! I've create an issue in the design system backlog for this. Would it be possible to get this PR in and make that update down the road? https://github.com/MetaMask/metamask-extension/issues/20635